services/__tests__/styles.test.js

```diff
@@ -131,10 +131,7 @@ describe('createStyleIcon', () => {
       path: '/styles/v1/:ownerId/:styleId/sprite/:iconId',
       method: 'PUT',
       params: { styleId: 'foo', iconId: 'bar' },
-      file: 'path/to/file.svg',
-      headers: {
-        'Content-Type': 'application/octet-stream'
```

Comment:
Before this change, we were setting `Content-Type: application/octet-stream` on this particular request in order to override the default `Content-Type: application/json`. The `octet-stream` value was not important in itself, I guess: it was just to override. So now that `application/json` will not be set (because there is no `body` for this request), that header is gone from the test. Make sense?

Reply:
Gotcha! 👌